src/angular/ts/tsComponentHandler.ts

```diff
@@ -65,7 +66,9 @@ export class TSComponentHandler {
       throw new Error("No class declaration found");
     }
     if (classDeclarations.length > 1) {
-      console.warn(`In this file are more that one class declarations`);
+      getLogger()
+        .getChildLogger({ label: "typescript" })
```

Comment:
Hier frage ich mich ob das Label sprechend genug ist. Weiß man wenn man das Label "typescript" ließt, dass sich der Fehler in dieser Datei befindet? Meine Überlegung wäre das Label vielleicht "TSComponentHandler" zu nennen :)

Reply:
oder wie du es an andere Stellen auch gemacht hast, den Methodenamen "findComponentReference"

Reply:
Es gibt eine einstellung mit der man genau zurückgeben kann in welcher Datei und welcher Zeile Code der Log erstellt wurde. Daher ist es villeicht overkill das immer mit rein zu schreiben. Bin mir selbst nicht sicher was hier der richtige Weg ist.

Also, immer genau sagen wo es erstellt wurde vs allgemein halten, weil man eh sehr schnell findet wo eine log message her kommt anhand des texts.

Reply:
Alles klar :)